x-pack/legacy/plugins/monitoring/public/components/table/eui_table_ssp.js

```diff
@@ -26,6 +26,11 @@ export function EuiMonitoringSSPTable({
     index: pagination.pageIndex,
     size: pagination.pageSize,
   });
+
+  if (!pagination.totalItemCount) {
+    pagination.totalItemCount = (items && items.length) || 0;
```

Comment:
Was this a separate issue you noticed?

Reply:
Yeah, this is another thing I noticed that I think was also introduced by EUI, kinda similar to: https://github.com/elastic/kibana/issues/39739

Reply:
Sounds good